docs/reference/api/hooks/contexts.rst

```diff
@@ -6,6 +6,6 @@ Contexts
 
 Contexts are a feature of the hook-based extension system in Tutor, which allows us to keep track of which components of the code created which callbacks. Contexts are very much an internal concept that most plugin developers should not have to worry about.
 
-.. autofunction:: tutor.hooks.contexts::enter
+.. autofunction:: tutor.core.hooks.contexts::enter
 
-.. autoclass:: tutor.hooks.contexts.Context
+.. autoclass:: tutor.core.hooks.Context
```

Comment:
This page appears under the catalog docs, so I think we'd want to show the Contexts catalog class, not the Context core type class.

```suggestion
.. autoclass:: tutor.hooks.Contexts
```

Reply:
Hmmm not quite. This page will actually appear in the "Hook types" section. The `Contexts` class is indeed documented in catalog.rst.